src/borg/helpers.py

```diff
@@ -29,6 +29,7 @@
 from operator import attrgetter
 from string import Formatter
 from shutil import get_terminal_size
+from posixpath import normpath
```

Comment:
rather use import posixpath here to make it clearer below that normpath is from there and not from os.path?